packages/twenty-front/src/modules/ui/feedback/snack-bar-manager/hooks/useSnackBar.ts

```diff
@@ -29,6 +30,15 @@ export const useSnackBar = () => {
     ({ set }) =>
       (newValue) =>
         set(snackBarInternalScopedState({ scopeId }), (prev) => {
+          if (
```

Comment:
Ah, this makes sense now! I didn't realize the issue was rooted at the snackbar level. I've been banging my head trying to implement workarounds: avoiding useEffect, steering clear of useRef for state, and adding guards. Thanks for catching this!